pandas/tests/io/parser/test_encoding.py

```diff
@@ -152,14 +152,17 @@ def test_binary_mode_file_buffers(
 
     with open(fpath, mode="r", encoding=encoding) as fa:
         result = parser.read_csv(fa)
+        fa.seek(0)
     tm.assert_frame_equal(expected, result)
 
     with open(fpath, mode="rb") as fb:
         result = parser.read_csv(fb, encoding=encoding)
+        fb.seek(0)
     tm.assert_frame_equal(expected, result)
 
     with open(fpath, mode="rb", buffering=0) as fb:
         result = parser.read_csv(fb, encoding=encoding)
+        fb.seek(0)
```

Comment:
is this necessary?  shouldnt the file be closed after this anyway?  same above

Reply:
I added it to make sure that `read_csv` doesn't close the file handle `assert not fb.closed` might be clearer